src/main/resources/db/migration/V1_3__create_table_behandler_dialogmelding_bestilling.sql

```diff
@@ -0,0 +1,21 @@
+CREATE TABLE BEHANDLER_DIALOGMELDING_BESTILLING
+(
+    id            SERIAL PRIMARY KEY,
+    uuid          VARCHAR(50) NOT NULL UNIQUE,
+    behandler_id  INTEGER NOT NULL REFERENCES BEHANDLER_DIALOGMELDING (id) ON DELETE RESTRICT,
+    personident   VARCHAR(11) NOT NULL,
```

Comment:
burde vi prefikse personident med arbeidstaker_ (for det er vel det det er?) for å gjøre det tydeligere?

Reply:
Ja, kan gjøre det.